### DISCUSSION TITLE
Lower Confidence Bound

### DISCUSSION THREAD

**Discussion by nikitrian** [meta-pytorch/botorch#1488] [Q&A] [answered]
Hello,

I have a minimization problem and I would like to use Lower Confidence Bound as the acquisition function. I see that Upper Confidence Bound is available in Botorch. If I set `maximize=False` in UCB (e.g `acquisition = UpperConfidenceBound(gp, beta=10, maximize=False`), would that give me the LCB? Or do I need to specify a custom acquisition function for LCB?

**Comment by saitcakmak** [comment, marked as answer]
Hi @nikitrian. Setting `maximize=False` will give you the negated LCB, i.e., it will return - mean + beta * std (see https://github.com/pytorch/botorch/blob/main/botorch/acquisition/analytic.py#L336-L339). If you then maximize this, e.g., using `optimize_acqf`, the candidate you get would be the point minimizing the LCB.

**Reply by nikitrian** [reply]
Thanks for your reply @saitcakmak. This makes sense, but I am not sure how to implement the maximization you are mentioning. 
```
    candidate = optimize_acqf(
        acquisition, bounds=bounds, q=q, num_restarts=50, raw_samples=1024,
    )
```
This is how I optimize my acquisition function. Is there a way to set this to maximization?
and this is how I set the acquisition function:

`acquisition = lambda gp: UpperConfidenceBound(gp, beta=1000, maximize=False)`

**Reply by saitcakmak** [reply]
`optimize_acqf` will maximize the given acquisition function, so that part seems correct to me.

> acquisition = lambda gp: UpperConfidenceBound(gp, beta=1000, maximize=False)

This pattern looks a bit odd to me. This will make acquisition require a GP as input, which would not work in `optimize_acqf`. Why not use `acquisition = UpperConfidenceBound(gp, beta=1000, maximize=False)` or something like
```
def get_acqf(gp):
    return UpperConfidenceBound(gp, beta=1000, maximize=False)

acquisition = get_acqf(gp)
```

You'll want the acquisition input to `optimize_acqf` to be an `AcquisitionFunction` (`UpperConfidenceBound` in this case) object.

**Reply by nikitrian** [reply]
@saitcakmak thanks a lot for your help. The implementation I have works well with expected improvement and I don't have any issues. The lower confidence bound seems to be working now as well.
